assets/scripts/dialogs/AnalyticsDialog.jsx

```diff
@@ -165,6 +174,18 @@ function AnalyticsDialog (props) {
               </p>
             </div>
             <div className="dialog-actions">
+              <label>
+                <input
+                  type="checkbox"
+                  checked={isVisible}
+                  onClick={toggleVisible}
+                />
```

Comment:
Interacting with this input element causes this warning in React:

> Warning: A component is changing an uncontrolled input of type checkbox to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components

Also, let's use our checkbox UI here: https://github.com/streetmix/streetmix/blob/master/assets/scripts/ui/Checkbox.jsx

You can see an example of its usage in the "Save to image dialog": https://github.com/streetmix/streetmix/blob/master/assets/scripts/dialogs/SaveAsImageDialog.jsx